src/lib/ng-package-format/entry-point.ts

```diff
@@ -66,7 +66,7 @@ export class NgEntryPoint {
 
     if (this.secondaryData) {
       primaryDestPath = this.secondaryData.primaryDestinationPath;
-      secondaryDir = path.basename(this.secondaryData.destinationPath);
+      secondaryDir = path.relative(primaryDestPath, this.secondaryData.destinationPath);
```

Comment:
Oops, if you have something like `core/testing` with basepath it will become `testing`

Reply:
It was part of the APF as before we had a flat dist structure